src/applications/Applications.ExtractImages/ExtractionMessageSender.cs

```diff
@@ -63,7 +66,7 @@ IConsoleInput consoleInput
             _nonInteractive = cliOptions.NonInteractive;
         }
 
-        public void SendMessages(string absoluteExtractionDir, ExtractionKey extractionKey, List<string> idList)
+        public void SendMessages(ExtractionKey extractionKey, List<string> idList)
```

Comment:
If you call this method multiple times does it overwrite the job file? For example if down the line we decided that `ExtractImagesHost` had to chunk up the file as it read it?  I can see the current implementation is all at once but I think we should deal with it being called multiple times.  It looks like there is even a setting `MaxIdentifiersPerMessage` implying that in future we might be sending multiple messages.

```csharp
var parser = new CohortCsvParser(_fileSystem);
(ExtractionKey extractionKey, List<string> idList) = parser.Parse(_csvFilePath);

_extractionMessageSender.SendMessages(extractionKey, idList);
```

Reply:
If it's called multiple times then yes the job ID file will be overwritten with a newly generated GUID. It should only ever be called once though; we already chunk up the input file into multiple messages here (see the `ermList` definition which calls `Chunk`).

`ExtractImagesHost` already checks that the extraction directory doesn't exist, but I can add a check for the jobId file if you think it's worth it?